unittest/test_activation_bounds.cpp

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2019-2023, LAAS-CNRS, New York University, Max Planck
+// Gesellschaft
+//                          University of Edinburgh, INRIA
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#define BOOST_TEST_NO_MAIN
+#define BOOST_TEST_ALTERNATIVE_INIT_API
+
+#include "crocoddyl/core/activations/quadratic-barrier.hpp"
+#include "unittest_common.hpp"
+
+using namespace boost::unit_test;
+using namespace crocoddyl::unittest;
+
+//----------------------------------------------------------------------------//
+void test_activation_bounds_with_infinity() {
+  Eigen::VectorXd lb(1);
+  Eigen::VectorXd ub(1);
+  double beta;
+  beta = 0.1;
+  lb[0] = 0;
+  ub[0] = std::numeric_limits<double>::infinity();
+
+  Eigen::VectorXd m =
+      0.5 * (lb + Eigen::VectorXd::Constant(
+                      lb.size(), std::numeric_limits<double>::max()));
+  Eigen::VectorXd d =
+      0.5 * (Eigen::VectorXd::Constant(lb.size(),
+                                       std::numeric_limits<double>::max()) -
+             lb);
+  crocoddyl::ActivationBounds bounds(lb, ub, beta);
+  BOOST_CHECK(bounds.lb != m - beta * d);
+}
+
+bool register_test() {
```

Comment:
When moving this into `test_activation.cpp`, please name this function as
```suggestion
bool register_bounds_unit_test() {
```

Reply:
Hello @cmastalli, do you mean, move the new test into the existing `test_activations.cpp` file or create a new test named, `test_activation.cpp`? Further, in the previous comment you mention, `test_activate.cpp`? If a new test is to be created which of it would be the name?

Reply:
Yes, I meant `test_activations.cpp`. I apologize for the typos.